### DISCUSSION TITLE
Feature request - Shared cursor in trend panel

### DISCUSSION THREAD

**Discussion by selvavm** [grafana/grafana#71597] [Dashboards, Panels & Visualizations]
Trend panel right now don't have shared cursor. It would be beneficial if it supports the same. 

I am not able to use timeseries panel since it refreshes on zoom and my usecase demands quicker refresh. This is supported in trend panel but unfortunately trend panel don't support shared panel which is also must for us

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82610. Feel free to continue the discussion around this there. Thank you!
